src/AppBundle/Repository/ChangeLogItemRepository.php

```diff
@@ -0,0 +1,13 @@
+<?php
```

Comment:
Hva gjør denne filen?

Reply:
Tror denne kan slettes siden det eksisterer en repository for ChangeLogItem her:  src/AppBundle/Entity/Repository/ChangeLogItemRepository.php